terragrunt/aws/load_balancer/load_balancer.tf

```diff
@@ -0,0 +1,40 @@
+# 
+# Application Load Balancer for ECS
+#
+
+resource "aws_lb" "cra_upd_ecs_alb" {
+  load_balancer_type = "application"
+  internal           = true
+  idle_timeout       = 300
+  subnets            = var.vpc_private_subnet_ids
+  security_groups    = [aws_security_group.cra_upd_loadbalancer_sg.id]
+}
+
+resource "aws_lb_target_group" "cra_upd_ecs_lb_target_group" {
+  port                              = 9000
+  protocol                          = "HTTP"
+  target_type                       = "ip"
+  vpc_id                            = var.vpc_id
+  load_balancing_cross_zone_enabled = false
+
+  deregistration_delay = 30
+
+  health_check {
+    path                = "/api/_healthcheck"
```

Comment:
Do you. have the healthcheck function defined in your code?

Reply:
Yup.

Reply:
Great. Thanks for confirming.